### ISSUE TITLE
Full Tanoa/Altis Compatibility

### ISSUE THREAD

**Issue by BoGuu** [AsYetUntitled/Framework#53]
Should we update the houses/containers SQL to support the input of the worldName, thus allowing people to use the same database for two maps? They ofcourse work independently, but if you ran both a Tanoa and Altis server off the same database, it would try incorrectly assign non-existent houses.

This would allow a community to run both maps, and allow synced persistence across them.

Thoughts?

**Comment by jcbjoe** [comment]
Yeah you could have 2 columns in the database, Tanoa Houses and Altis Houses and depending on the worldName retrieve

**Comment by setoy** [comment]
Yep, a `worldName` column would be perfect.

**Comment by tkcjesse** [comment]
Or just let people create a new house table and figure it out themselves. Because what if they plan on doing 2 Tanoa servers instead of Altis & Tanoa... Idk, I'd just let them figure out how to change a few sql statements to use a new table for the new server...

**Comment by jcbjoe** [comment]
Surely It wouldn't matter? Most servers have two Altis servers so they can accept more players. The houses are shared anyway between server. The point of using the same database is so that players data is consistent

**Comment by tkcjesse** [comment]
No... the purpose that is being brought up here is to not share the same house on separate servers. Why do that...

**Comment by jcbjoe** [comment]
Fair play, I was just looking at what all servers do atm as they dont have the option to not share housing. However this could be a option to toggle, i still think most community would want sharing of homes. The servers big enough to afford and maintain 2 servers are the ones where server #1 lets say is always full so the second is used as a overflow allowing players to still access there house even tho they cant get on there main one.

**Comment by BoGuu** [comment]
There is plenty of houses on Tanoa to support 2 Tanoa servers. The idea of this is purely aimed at servers wishing to open a second server to support a larger player base. Which isn't currently too feasible (without changes) on two different maps.

Nothing is stopping people from using two different schema ofcourse.

**Comment by dbenson24** [comment]
I really think that adding a worldName column is unnecessary, If someone wants to run two different servers, it's really not that hard to have them point to different housing tables in the same database

**Comment by jcbjoe** [comment]
As i said before we could have it so it auto detects the world name and then chooses the appropriate table. I think the thing were trying to do here is make the mission/framework easier for the end user to use.

**Comment by BoGuu** [comment]
@dbenson24 

The point is, it's two changes, each which are a couple of words, just for ease. Won't affect anything at all. Just nice to give people the option instead of faffing with tables etc.

**Comment by Big-Daddy-Grimm** [comment]
i would like to see this option implemented
